<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bExport">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Data</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="exportDateFrom">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>194</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="exportDateTo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>194</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="eraseDateFrom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>194</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bErase">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase data</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="eraseDateTo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>194</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
